### DISCUSSION TITLE
如何处理消息中的富文本

### DISCUSSION THREAD

**Discussion by luoyuemin** [simple-robot/Discussions#142] [Q&A]
### 涉及的编程语言

Java

### 涉及的组件库

mirai (QQ)

### 疑问描述

消息中含有图片文字，我如何把图片下载到本地上啊求java代码案列~~~呜呜呜

**Comment by ForteScarlet** [comment]
参考 https://github.com/orgs/simple-robot/discussions/15

`Image` 参考其API文档 https://docs.simbot.forte.love/main/simbot-api/love.forte.simbot.message/-image/index.html ，可通过 `resource` 属性获取资源，资源可用于下载，或判断为 [URLResource](https://docs.simbot.forte.love/main/simbot-api/love.forte.simbot.resources/-u-r-l-resource/index.html) 后获取 `url` 地址。
